packages/datatrak-web/src/layout/UserMenu/ProjectSelectModal.tsx

```diff
@@ -8,9 +8,78 @@ import { Paper } from '@material-ui/core';
 import { ProjectSelectForm } from '@tupaia/ui-components';
 import { RequestProjectAccess } from '../../features';
 import { useCurrentUserContext, useEditUser, useProjects } from '../../api';
-import { Modal } from '../../components';
+import { Modal, SlideTransition } from '../../components';
 
-const Wrapper = styled(Paper)`
+const StyledModal = styled(Modal)`
+  //  The mobile styles are specific to the project select modal in datatrak-web so they are included here
+  //  instead of in the ui-components select list component
+  ${({ theme }) => theme.breakpoints.down('sm')} {
+    .MuiPaper-root {
+      height: 100%;
+      background: ${({ theme }) => theme.palette.background.default};
+
+      > div {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        height: 100%;
+
+        > div {
+          max-height: 100%;
+        }
+      }
+    }
+
+    .list-wrapper {
+      border: none;
+      border-radius: 0.625rem;
+    }
+
+    h2.MuiFormLabel-root {
+      color: ${({ theme }) => theme.palette.text.secondary};
+    }
+
+    // Hide the close button on mobile
+    .MuiButtonBase-root.MuiIconButton-root {
+      display: none;
+    }
+
+    // Select list
+    .MuiList-root {
+      border-radius: 0.625rem;
+      background: ${({ theme }) => theme.palette.background.paper};
+      padding-inline: 1rem;
+      padding-block: 0.3rem;
+
+      > li {
```

Comment:
Does `.MuiListItem-root` apply here? If so, could let the nesting one level shallower here for readability

Will defer to you, though